samples/web/README.md

```diff
@@ -0,0 +1 @@
+Please refer to [Readme.md](./nextjs/README.md) under the nextjs sample folder.
```

Comment:
Let's move the /next.js/README.md to this folder and remove all README.md files from each sample folders. This way we do not have to redirect everyone to the correct readme file.